webpack/JobWizard/steps/form/ResourceSelect.js

```diff
@@ -50,26 +52,43 @@ export const ResourceSelect = ({
         description={__('Please refine your search.')}
       >
         {sprintf(
-          __('You have %s results to display. Showing first %s results'),
-          response.subtotal,
+          __('You have more results to display. Showing first %s results'),
           maxResults
         )}
       </SelectOption>,
     ];
   }
   selectOptions = [
     ...selectOptions,
-    ...Immutable.asMutable(response?.results || [])?.map((result, index) => (
-      <SelectOption key={index + 1} value={result.id}>
-        {result.name}
-      </SelectOption>
-    )),
+    ...Immutable.asMutable(response?.results || [])
+      ?.slice(0, maxResults)
+      ?.map((result, index) => (
+        <SelectOption key={index + 1} value={result.id}>
+          {result.name}
+        </SelectOption>
+      )),
   ];
 
   const onSelect = (event, selection) => {
     setSelected(selection);
     setIsOpen(false);
   };
+  const onFilter = (_, value) => {
+    if (!value) {
+      return selectOptions;
+    }
+    return selectOptions.filter(
+      o =>
+        typeof o.props.children === 'string' &&
+        o.props.children.toLowerCase().indexOf(value.toLowerCase()) > -1
```

Comment:
Results from API are being returned regardless of casing. This is to support case insensitivity in UI.